### PR TITLE
Update state-badge.ts: change border-radius for media_player

### PR DESCRIPTION
**Pull request by ildar170975**
[Discussion](https://github.com/home-assistant/frontend/discussions/16871)

before:
244960198-db480461-3946-4aec-812c-a2e31f091946

after:
244960311-a99d209b-6fa3-40e9-acaa-8f21a3f27387

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by matthiasdebaat** [comment]
Looks good 👍

File: src/components/entity/state-badge.ts [hunk 1]
```diff
@@ -140,6 +140,9 @@ export class StateBadge extends LitElement {
         if (domain === "update") {
           hostStyle.borderRadius = "0";
         }
+        if (domain === "media_player") {
+          hostStyle.borderRadius = "10%";
```
Comment (bramkragten) [inline src/components/entity/state-badge.ts:144]:
```suggestion
          hostStyle.borderRadius = "8%";
```

Comment (bramkragten) [reply src/components/entity/state-badge.ts:144]: To match our border radius usage in other places

Comment (ildar170975) [reply src/components/entity/state-badge.ts]: Good, can we merge?

Comment (bramkragten) [reply src/components/entity/state-badge.ts:144]: if you apply the suggestion 😄 👍

**Review by bramkragten — APPROVED** [review]

**Pull request merged**